light-node/tests/acceptance.rs

```diff
@@ -45,46 +43,21 @@ fn start_no_args() {
 #[test]
 #[ignore]
 fn start_with_args() {
-    let mut runner = RUNNER.clone();
-    let mut cmd = runner
-        .args(&["start", "acceptance", "test"])
-        .capture_stdout()
-        .run();
-
-    cmd.stdout().expect_line("Hello, acceptance test!");
-    cmd.wait().unwrap().expect_success();
+    todo!()
 }
 
 /// Use configured value
 #[test]
 #[ignore]
 fn start_with_config_no_args() {
-    let mut config = LightNodeConfig::default();
-    config.rpc_address = "localhost:26657".to_owned();
-    let expected_line = format!("Requesting from {}.", &config.rpc_address);
-
-    let mut runner = RUNNER.clone();
-    let mut cmd = runner.config(&config).arg("start").capture_stdout().run();
-    cmd.stdout().expect_line(&expected_line);
-    cmd.wait().unwrap().expect_success();
+    todo!()
 }
 
 /// Override configured value with command-line argument
 #[test]
 #[ignore]
 fn start_with_config_and_args() {
-    let mut config = LightNodeConfig::default();
-    config.rpc_address = "localhost:26657".to_owned();
-
-    let mut runner = RUNNER.clone();
-    let mut cmd = runner
-        .config(&config)
-        .args(&["start", "other:26657"])
-        .capture_stdout()
-        .run();
-
-    cmd.stdout().expect_line("Requesting from other:26657.");
-    cmd.wait().unwrap().expect_success();
+    todo!()
```

Comment:
Note that we should also capture running these tests against a fullnode in integration tests. Should also capture this in a followup issue.

Reply:
Do we have one for this?